src/sections/DevFeature2/index.tsx

```diff
@@ -51,16 +51,16 @@ export default function DevFeature2() {
             <img className="mx-auto lg:max-w-4xl" src="/transfer.gif" />
             <div className="mx-auto mt-2  flex max-w-4xl justify-between gap-2">
               <GiantButton
-                href="https://docs.buildwithsygma.com/sdk/quickstart/examples/erc20/substrate-evm-example"
```

Comment:
the link is actually: https://docs.buildwithsygma.com/sdk/quickstart/substrate-token-transfer